client/dapp_chain_rpc_client.go

```diff
@@ -485,10 +485,11 @@ func (c *DAppChainRPCClient) CommitDeployTx(
 func (c *DAppChainRPCClient) CommitMigrationTx(
 	from loom.Address,
 	signer auth.Signer,
-	id uint32,
+	id uint32, inputparams []byte,
```

Comment:
`inputparams` -> `input`
And this file doesn't appear to be auto-formatted

Reply:
Done.